### ISSUE TITLE
Question about equation 2

### ISSUE THREAD

**Issue by supersyz** [kxhit/EscherNet#24]
Thanks for your nice work！
![image](https://github.com/user-attachments/assets/fe81a8ef-2d6d-4e48-9996-36d0e9756f85)
Previous works like One-2-3-45 mentioned that the relative camera pose is related to the absolute camera pose, i.e., the elevation. And if the absolute elevation is not considered, the reconstructed 3D shapes could be terrible. 
It seems that equation 2 does not take this into consideration.  Does it implicitly takes this into account, or it ignores it ?
Hope for your reply！

**Comment by kxhit** [comment]
No, EscherNet is designed to handle any 6DoF/4DoF poses, we don't need to estimate the elevation of the input image.
